### DISCUSSION TITLE
Supabase can't connect to Docker

### DISCUSSION THREAD

**Discussion by PauloCesar1218** [supabase/supabase#9706] [Questions]
I'm an M1 pro mac so I installed docker for the M1 chip and installed supabase with homebrew, but when I try `supabase start` I get the following error: 
![Captura de Tela 2022-10-19 às 15 56 59](https://user-images.githubusercontent.com/43884265/197011267-70564a64-62ec-411e-92f1-3d7889f96bf9.png)

Does anyone know what might be the problem?

**Comment by benjaminbialy** [comment]
Hey, running into the same problem. Did you find a fix?
